### ISSUE TITLE
Ignore Migrations before a certain Timestamp

### ISSUE THREAD

**Issue by keesbriggs** [ankane/strong_migrations#83]
As a developer adding StrongMigrations to my long-existing application, I would like the ability to only use StrongMigrations moving forward, and not on historical migrations.

`StrongMigrations.start_after` does not appear to work.

**Comment by ankane** [comment]
Hey @keesbriggs, check out the readme.

**Comment by keesbriggs** [comment]
@ankane I did. And I don't see where the functionality is implemented in the code. And it does not work...

**Comment by bheemreddy181** [comment]
take a look at this https://github.com/ankane/strong_migrations#existing-migrations

**Comment by keesbriggs** [comment]
Yes, understood. However it does not work. Can you point to the lines of code where this functionality is implemented?

**Comment by keesbriggs** [comment]
In my config/initializers/strong_migrations.rb: 
```
StrongMigrations.start_after = 20190618211220
```

However when I run migrations I see: 
```
...
== 20130807212114 AddImageUrlToMessages: migrating ============================
-- add_column(:messages, :image_url, :string)
   -> 0.0005s
== 20130807212114 AddImageUrlToMessages: migrated (0.0005s) ===================

rake aborted!
StrongMigrations::UnsafeMigration: 
=== Dangerous operation detected #strong_migrations ===

ActiveRecord caches attributes which causes problems
when removing columns. Be sure to ignore the column:

class Message < ActiveRecord::Base
  def self.columns
    super.reject { |c| ["subject"].include?(c.name) }
  end
end

Deploy the code, then wrap this step in a safety_assured { ... } block.

class ActiveRecord::Migration < ActiveRecord::Migration
  def change
    safety_assured { remove_column :messages, :subject }
  end
end

/Users/keesbriggs/.rvm/gems/ruby-2.3.1/gems/activerecord-4.2.11.1/lib/active_record/migration.rb:416:in `method_missing'
/Users/keesbriggs/Sites/wonolo-rails/db/migrate/20130807222608_remove_subject_from_message.rb:4:in `block (2 levels) in <class:RemoveSubjectFromMessage>'
```

**Comment by keesbriggs** [comment]
I reviewed the code and I see this variable used here:
```
https://github.com/ankane/strong_migrations/blob/master/lib/strong_migrations.rb#L15
```
However I do not see this attribute used anywhere else in any of the files.

**Comment by keesbriggs** [comment]
```
Searching 25 files for "start_after"

/Users/keesbriggs/Sites/strong_migrations/CHANGELOG.md:
   49: - Added `start_after` option

/Users/keesbriggs/Sites/strong_migrations/lib/strong_migrations.rb:
   12:     attr_accessor :auto_analyze, :start_after, :checks, :error_messages, :target_postgresql_version
   15:   self.start_after = 0

/Users/keesbriggs/Sites/strong_migrations/lib/strong_migrations/checker.rb:
  191:       version && version <= StrongMigrations.start_after

/Users/keesbriggs/Sites/strong_migrations/README.md:
  480: StrongMigrations.start_after = 20170101000000

/Users/keesbriggs/Sites/strong_migrations/test/test_helper.rb:
   17: StrongMigrations.start_after = 20170101000000

6 matches across 5 files
```

**Comment by keesbriggs** [comment]
OK - I cloned the repo and I see `lib/strong_migrations/checker.rb` in my codebase.

However, when I look in 
```
/Users/keesbriggs/.rvm/gems/ruby-2.3.1/gems/strong_migrations-0.3.1/lib/strong_migrations.rb
```
...that file is not present. I believe it is missing from the gem.

**Comment by keesbriggs** [comment]
@bheemreddy181 ^^

**Comment by bheemreddy181** [comment]
I am using version 0.3.1 for my app and it works fine for me. Is this the same version you are using?

**Comment by keesbriggs** [comment]
I am using 0.3.1 as well and when I uninstall/install the gem, it remains un-included.

**Comment by keesbriggs** [comment]
Looks like that feature wasn't available in 0.3.1. When I switch to the v0.3.1 tag, that file is missing. Thanks, I'll adjust accordingly.

**Comment by ankane** [comment]
For future readers, this feature was introduced in 0.1.9.

@keesbriggs Please use Stack Overflow if you have additional questions about it.
